src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/service/RerouteQueueService.java

```diff
@@ -228,15 +230,35 @@ public int compare(FlowThrottlingData throttlingDataA, FlowThrottlingData thrott
             int priorityB = throttlingDataB.getPriority() == null ? defaultFlowPriority : throttlingDataB.getPriority();
             Instant timeCreateA = throttlingDataA.getTimeCreate();
             Instant timeCreateB = throttlingDataB.getTimeCreate();
+            PathComputationStrategy pathComputationStrategyA = throttlingDataA.getPathComputationStrategy();
```

Comment:
As for me it should be separate comparator joined to existing via `thenComparing`.

Reply:
Reworked